docs/Code-Reviews.md

```diff
@@ -0,0 +1,161 @@
+# Helpful code reviews
+
+This document presents some of the principles behind how and why we do code reviews, as well as collecting some useful bits of advice to make code reviews more effective and enjoyable for everyone.
+
+It is not meant to be an exhaustive guide to carrying out reviews - there are already many online resources on this subject. This document incorporates some of their ideas (especially from Gergely Orosz’s excellent [Good Code Reviews, Better Code Reviews](https://blog.pragmaticengineer.com/good-code-reviews-better-code-reviews/)) but expands on a few topics of particular importance for our teams.
+
+## Why review code?
+
+Reviewing code can sometimes help to catch bugs earlier in the development life cycle when [they](http://thklein.com/cost-of-defect/) [*may*](http://thklein.com/cost-of-defect/) [be cheaper to fix](http://thklein.com/cost-of-defect/). Even then, it is not realistic to expect most defects to be caught this way. In large, distributed teams like ours, the main objective of code reviews is *communication*.
```

Comment:
```suggestion
Reviewing code can sometimes help to catch bugs earlier in the development life cycle when [they *may* be cheaper to fix](http://thklein.com/cost-of-defect/). Even then, it is not realistic to expect most defects to be caught this way. In large, distributed teams like ours, the main objective of code reviews is *communication*.
```

Not sure if the separation of links is intentional, as they point to the same page.

Reply:
No that was an error from exporting from Dropbox Paper. Thanks Ruth :-)